yarn.lock

```diff
@@ -2,28 +2,28 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.12.11", "@babel/code-frame@^7.0.0":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz"
+  resolved "https://caminc.jfrog.io/caminc/api/npm/default/@babel/code-frame/-/code-frame-7.12.11.tgz"
```

Comment:
Don't change our registry to this `caminc.jfrog.io`, we don't use this. This is why causes this CI error:

![image](https://github.com/node-casbin/mongoose-adapter/assets/3787410/4e475431-75c9-4cdd-a0b3-6096b9044b0f)


Keep using `registry.npmjs.org` or latest `registry.npmjs.com`

Reply:
sorry, I have fixed yarn.lock.